{"name": "用户输入运算数据a、b，a或b大小超出范围", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'data/add_data.yaml'", "trace": "request = <SubRequest 'cmd_option' for <Function test_div[10-5-2]>>\n\n    @pytest.fixture(scope=\"session\", autouse=True)\n    def cmd_option(request):\n        \"\"\"\n        查看指定测试用例的测试数据\n        :param request:  参数化\n        :return:\n        \"\"\"\n        global data_path\n        my_file_path = request.config.getoption(\"--show\", default=\"add_data\")\n        if my_file_path == \"add_data\":\n            data_path = r\"data/add_data.yaml\"\n        elif my_file_path == \"range_in_add_data\":\n            data_path = r\"data/add_data_range_in.yaml\"\n        elif my_file_path == \"range_out_data\":\n            data_path = r\"data/data_range_out.yaml\"\n        elif my_file_path == \"type_error_data\":\n            data_path = r\"data/data_type_error.yaml\"\n        elif my_file_path == \"div_data\":\n            data_path = r\"data/div_data.yaml\"\n        elif my_file_path == \"rang_in_div_data\":\n            data_path = r\"data/div_data_rang_in.yaml\"\n        elif my_file_path == \"zero_division_error\":\n            data_path = r\"data/zero_division_error.yaml\"\n    \n>       with open(data_path, \"r\", encoding=\"utf-8\") as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'data/add_data.yaml'\n\n..\\conftest.py:83: FileNotFoundError"}, "description": "\n        测试add()函数的数据范围内参数\n        :param a: 输入集合{98，98.9，-98.9，-98}中的任意一个数据 a；\n        :param b: 输入集合{98，98.9，-98.9，-98}中的任意一个数据 b；\n        :param expected: 程序输出a加b的结果；\n        :return: 返回 a加 b的结果；\n        ", "parameters": [{"name": "a", "value": "98"}, {"name": "b", "value": "98"}, {"name": "expected", "value": "196"}], "start": 1715221620711, "stop": 1715221620711, "uuid": "9821d046-36ab-4d74-904a-451921bfb43c", "historyId": "3e9d48129aa560a053784ebaf6332df8", "testCaseId": "aa5f27629e98e9b9bb52833a1d5b0a3c", "fullName": "test_case.test_case_03.TestCase#test_add_data_range_in", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "tag", "value": "test_data_range"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_03"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "wangmin"}, {"name": "thread", "value": "22052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_03"}], "links": [{"type": "tms", "url": "http://192.168.40.134:8080/projects/CACULATOR/issues/CACULATOR-5?filter=allissues", "name": "用例管理系统"}]}